src/ConcreteOperations/volume.jl

```diff
@@ -0,0 +1,34 @@
+export volume
+
+"""
+    volume(H::AbstractHyperrectangle)
+
+Return the volume of a hyperrectangular set.
+
+### Input
+
+- `H` -- hyperrectangular set
+
+### Output
+
+The (Euclidean) volume of ``H``.
+
+### Algorithm
+
+The volume of the hyperrectangle ``H`` with vector radius ``r`` is
```

Comment:
```suggestion
The volume of the ``n``-dimensional hyperrectangle ``H`` with vector radius ``r`` is
```
(because `n` is used in the formula)